doc/ITEM_SPAWN.md

```diff
@@ -244,20 +244,20 @@ Instead of a full JSON object, one can also write a JSON array. The default subt
 You can test your item groups in the game:
 
 1. Load a game and call the debug menu
-    > *TIP* (If a key isn't bound to the debug menu or you forgot it, use <kbd>ESC</kbd> > <kbd>1</kbd>)
+    > *TIP*: If a key isn't bound to the debug menu or you forgot it, use <kbd>ESC</kbd> > <kbd>1</kbd>.
 
 2. Choose "Test Item Group".
 
 3. Select the item group you want to debug.
 
-    The game will then spawn items in that group 100 times and count the spawned items. They'll then be displayed, sorted by their frequency.
+    The game will then spawn items in that group 100 times and count the spawned items.  They'll then be displayed, sorted by their frequency.
 
-    > *TIP*: You can filter anything in the debug menu using <kbd>/</kbd>
+    > *TIP*: You can filter anything in the debug menu using <kbd>/</kbd>.
 
-You should not add items to the item group `EMPTY_GROUP`. This group can be used when the game requires a group id, but you don't want to spawn any items there. The group will never spawn items.
+You should not add items to the item group `EMPTY_GROUP`.  This group can be used when the game requires a group id, but you don't want to spawn any items there; it will never spawn items.
 
 #### SUS
 
-When adding items to item groups, attempt to locate or create **SUS item groups.** SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given piece of storage furniture. SUS stands for "specific use storage." One of the aims of organizing item groups into specific use storage is to promote reusable tables that can be maintained and extended.
+When adding items to item groups, attempt to locate or create **SUS item groups.**  SUS item groups are collections that contain a reasonable realistic distribution of items that might spawn in a given piece of storage furniture.  SUS stands for "specific use storage."  One of the aims of organizing item groups into SUS groups is to promote reusable tables that can be maintained and extended.
 
-You can find SUS item groups at /data/json/itemgroups/SUS
+You can find existing SUS item groups at [/data/json/itemgroups/SUS](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/itemgroups/SUS/).
```

Comment:
Should a note be made that, even if not using SUS groups, one should place item groups in data/json/itemgroups if they aren't only usable in a single file? Putting them there makes them much easier to find!